### PR TITLE
UI Enhancement and Minor Bug Fix

### PR DESCRIPTION
**Pull request by jewiteo**
closes #144 #150 #133 

Made minor changes to the UI, namely showing what week/period the schedule is currently displaying and stylistic changes.

Fixes bug where associated contacts to meetings were connected by formatting the flowPane.

image

### CODE REVIEW

File: src/main/resources/view/WeeklyCalendarPanel.fxml [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.control.ListView?>
-<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.control.*?>
```
Comment (tata32000) [inline src/main/resources/view/WeeklyCalendarPanel.fxml:3]: Should we use wildcard import here?

**Review by tata32000 — CHANGES_REQUESTED** [review]

**Comment by LimKaiWei** [comment]
for people with long names is it possible to truncate with `...` to fit in calendar card?
image

File: src/main/resources/view/WeeklyCalendarPanel.fxml (hunk 1 above)
Comment (jewiteo) [reply src/main/resources/view/WeeklyCalendarPanel.fxml]: Nope, will explicitly state it. Thanks for the catch

**Review by LimKaiWei — APPROVED** [review]
LGTM

File: src/main/resources/view/PersonListCard.fxml [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
 
-<?import javafx.geometry.Insets?>
-<?import javafx.scene.control.Label?>
-<?import javafx.scene.layout.ColumnConstraints?>
-<?import javafx.scene.layout.FlowPane?>
-<?import javafx.scene.layout.GridPane?>
-<?import javafx.scene.layout.HBox?>
-<?import javafx.scene.layout.Region?>
-<?import javafx.scene.layout.VBox?>
+<?import javafx.geometry.*?>
```
Comment (tata32000) [inline src/main/resources/view/PersonListCard.fxml:3]: We shouldn't use wildcard here right?

**Review by tata32000 — CHANGES_REQUESTED** [review]

**Review by tata32000 — APPROVED** [review]

**Pull request merged**